ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/EngineConnectorConfig.kt

```diff
@@ -126,8 +128,9 @@ class EngineSSLConnectorBuilder(
     override var keyStore: KeyStore,
     override var keyAlias: String,
     override var keyStorePassword: () -> CharArray,
-    override val privateKeyPassword: () -> CharArray,
-    override val requiredClientCACertificates: List<X509Certificate> = emptyList()
+    override var privateKeyPassword: () -> CharArray,
 ) : EngineConnectorBuilder(ConnectorType.HTTPS), EngineSSLConnectorConfig {
     override var keyStorePath: File? = null
+    override var trustStore: KeyStore? = null
+    override var trustStorePath: File? = null
```

Comment:
Personally, I don't understand a useful use case with trustStore and trustStorePath. I don't know, why keyStorePath is available...

Reply:
I guess you can figure this out from keyStorePath usage or ask ktor team. Personally I believe it is there for convenience but of course there might be stronger incentives.